CHANGELOG.md

```diff
@@ -8,6 +9,12 @@
 - [v2.15.0-ck8s1](#v2150-ck8s1---2021-05-27)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v2.19.0-ck8s1 - 2022-06-13
```

Comment:
Don't you want to have the kubespray upgrade first? then you dont need to merge the fix into main later